### DISCUSSION TITLE
Unable to run Shadcn www locally

### DISCUSSION THREAD

**Discussion by Chukwu3meka** [shadcn-ui/ui#6039] [General]
After cloning Shadcn and installing packages it compiles successfully, I can view the home page, Blocks, Themes, Colors. etc. Once I try to open docs or components I get this error `Module not found: Can't resolve 'contentlayer/generated'`.

` ⨯ ./components/framework-docs.tsx:5:1
Module not found: Can't resolve 'contentlayer/generated'
  3 | import * as React from "react"
  4 | // import { allDocs } from "contentlayer/generated"
> 5 | import { allDocs } from "contentlayer/generated"
    | ^
  6 |
  7 | import { Mdx } from "./mdx-components"
  8 |

https://nextjs.org/docs/messages/module-not-found`

![image](https://github.com/user-attachments/assets/3f4a8018-114e-468f-ba24-de188d86c459)
![image](https://github.com/user-attachments/assets/f5460490-a8b8-4db4-9d8b-69335ff0e25f)

**Comment by Chukwu3meka** [comment]
I also get this warning after running `pnpm www:dev`

![image](https://github.com/user-attachments/assets/c380c012-fb24-438c-9a6d-b7b79046dca6)

**Reply by georgeeburt** [reply]
Have you installed contentlayer & next-contentlayer packages? You'll also have to create a content layer config file and add that to your `next.config.js` file.

**Reply by Chukwu3meka** [reply]
This would add additional depencies to package.json not previously included in the main branch. Also contentlayer.config.js exists

**Reply by Chukwu3meka** [reply]
@georgeeburt do you mind if I have a look at your branch

**Comment by Chukwu3meka** [comment]
Aside the initial `pnpm install` which did not work for me, I had to navigate to `/apps/www` and then run `pnpm install`
